### PR TITLE
Upgrade @rollup/plugin-node-resolve and rollup-plugin-terser

### PR DESCRIPTION
**Pull request by apasov**
Upgrade the plugins to their latest major versions:
@rollup/plugin-node-resolve to ^8.0.0
rollup-plugin-terser to ^6.0.0

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const path = require('path');
-const resolve = require('@rollup/plugin-node-resolve');
+import resolve from '@rollup/plugin-node-resolve';
```
Comment (lahmatiy) [inline rollup.config.js:2]: As I know `import` and `require()` can't be used in a single module. I would be surprised if this works ...

File: package.json [hunk 2]
```diff
@@ -42,15 +42,15 @@
...
+    "@rollup/plugin-node-resolve": "^8.0.0",
     "clap": "^2.0.1",
     "coveralls": "^3.0.9",
     "eslint": "^6.8.0",
     "json-to-ast": "^2.1.0",
     "mocha": "^5.2.0",
     "nyc": "^14.1.1",
     "rollup": "^1.32.1",
-    "rollup-plugin-terser": "^5.3.0"
+    "rollup-plugin-terser": "^6.0.1"
```
Comment (lahmatiy) [inline package.json:53]: 6.0 doesn't support rollup before 2.0, and currently rollup below 2.0 is used.
Also 6.0 has dropped support for Node < 10, but css-tree supports Node 8.0+ at the moment.

File: rollup.config.js (hunk 1 above)
Comment (apasov) [reply rollup.config.js:2]: They do work together, at least on the latest Node which is 14.3.0
The reason why I changed `require()` to `import` is that I got this error after upgrading the plugin:

```
$ rollup -c
[!] TypeError: resolve is not a function
TypeError: resolve is not a function
    at Object.<anonymous> (rollup.config.js:30:5)
    at Module._compile (internal/modules/cjs/loader.js:1200:30)
    at Object.require.extensions.<computed> [as .js] (node_modules\rollup\dist\shared\loadConfigFile.js:516:20)
    at Module.load (internal/modules/cjs/loader.js:1049:32)
    at Function.Module._load (internal/modules/cjs/loader.js:937:14)
    at Module.require (internal/modules/cjs/loader.js:1089:19)
    at require (internal/modules/cjs/helpers.js:73:18)
    at loadConfigFromBundledFile (node_modules\rollup\dist\shared\loadConfigFile.js:524:42)
    at getDefaultFromTranspiledConfigFile (node_modules\rollup\dist\shared\loadConfigFile.js:509:12)
    at loadConfigFile (node_modules\rollup\dist\shared\loadConfigFile.js:487:15)

error Command failed with exit code 1.
```

rollup is 2.10.9

File: package.json (hunk 2 above)
Comment (apasov) [reply package.json:53]: Ok, I see. So maybe this could be changed in the future.

**Comment by lahmatiy** [comment]
Looks like all works fine without PR changes.

**Pull request closed without merging**